uPortal-api/uPortal-api-rest/src/main/java/org/apereo/portal/rest/MarketplaceRESTController.java

```diff
@@ -165,4 +165,72 @@ public ModelAndView saveUserRating(
         return new ModelAndView(
                 "json", "rating", new MarketplaceEntryRating(Integer.parseInt(rating), review));
     }
+
+    /**
+     * Get all ratings done on portlets since daysBack ago.
+     *
+     * @param request
+     * @param daysBack - number of days back to search for portlet ratings
+     * @return
+     */
+    @RequestMapping(value = "/marketplace/{daysBack}/getRatings", method = RequestMethod.GET)
```

Comment:
:question: what if data from two weeks ago to last week is wanted?
Perhaps supporting a date range could future proof the API more?

Reply:
Christian - if you or others would like to add more end points please do so.  I'm done with this.

Reply:
I'd like to add `v5-1` to the URI.